### DISCUSSION TITLE
ALL Markdown Link Icons Showing Even when I'm NOT Hovering over it

### DISCUSSION THREAD

**Discussion by Dvidal7788** [community/community#67960] [Other Feature Feedback, Questions, & Ideas]
Is anybody else having an issue where the link icon for ALL of the markdown elements in your ReadMe file are showing even when you're not hovering over it? Only one should show at a time and it should be only when you're hovering your mouse over it....maybe its a just a GitHub problem, because I copied and pasted the exact text into a GitLab repo and it seems to work fine. Also, this seems to be happening in all my GitHub repos (which was not happening before and I haven't touched those repos in months), so its not like I messed up the markdown. I also now notice it on other people's repos too. I even cleared my browser's cache, tried another browser, even tried another computer and I still see it happening in mine and other people's repos, so this is very odd.

...Also, even when I revert back to an old commit of the readme, before this issue popped up, it still happens...

My mouse is nowhere near any of those elements, but the link icon is showing for all of them...
![image](https://github.com/community/community/assets/91298183/4b5dc8b8-65db-444e-a182-320d7a017276)

Same exact markdown (copied & pasted) on GitLab and the only link icon showing is for the one element that my mouse is near, which is how its supposed to work.
![image](https://github.com/community/community/assets/91298183/e2cba678-0739-4731-b499-f4e990cc6979)

You can see it here:
www.github.com/Dvidal7788/dataLynx

**Comment by Wolanet** [comment]
Same issue here, you can see it in my profile's README. Also for some reason after I refresh my profile, all those "links" become of blue color, like if they were actual links instead of headers.

Is it a bug? @Github staff , hello??

**Comment by SumanthSShet** [comment]


**Reply by Wolanet** [reply]
If the GitHub team is investigating it then I'll wait for a fix! Thanks for letting us know.

Btw, I'm not sure what you mean with "Use code with caution.", was that code you posted generated with Bard? I should assume that it might not work?

**Comment by Wolanet** [comment]
why did he delete the comment?
...

**Reply by Dvidal7788** [reply]
I don't know, but I pasted his original comment below (still had it in my email notifications). I couldn't get it to work though. I even tried some stuff here to include the CSS: https://stackoverflow.com/questions/51956361/custom-css-file-for-readme-md-in-a-github-repo

I think he said use with caution because GitHub doesn't actually support CSS, so using CSS can sometimes lead to issues or maybe just unknown behavior?.....

Original response: 
Yes, I am aware of the issue where the link icon for all of the markdown elements in a README file are showing even when you're not hovering over it. This is a known issue with GitHub Pages and is currently being investigated by the GitHub team.

In the meantime, there is a workaround that you can use to fix this issue. You can add the following CSS to your README file:

CSS
a {
text-decoration: none;
box-shadow: none;
}
Use code with caution. [Learn more](https://bard.google.com/faq#coding)
This will remove the link icon from all of the markdown elements in your README file.

To add this CSS to your README file, you can either:

Create a new file called README.md.css and add the CSS code above to it. Then, update your README file to include the following line:
Markdown
Use code with caution. [Learn more](https://bard.google.com/faq#coding) Add the CSS code above directly to your README file. Once you have added the CSS code, you will need to commit your changes and push them to your repository.

I hope this helps!

**Comment by Wolanet** [comment]
Yeah doing that didn't fix it. Also he linked Bard for some reason, I guess he just copy-pasted an AI answer. Idk.

This is really annoying, also not sure if it's just me but my README is basically a giant BLUE LINK, because every header I use turns into a blue link. What's weird is that this happens only when I refresh my profile page, if I go to edit my README (without actually doing any changes) and then go back, they turn back to being "white links".

How come no one seems to care about this issue which is "global" and affects all users? Can someone from github staff / support provide us with some explanation or info about it being fixed? 
That guy said it's a known issue and it's being investigated, but he then deleted his comment and he's not even part of github staff, honestly kinda weird.

If anyone from support or @GitHub staff could comment on this, it'd be very appreciated. Thanks.

**Comment by Ddhruv-IOT** [comment]
I am also facing the same issue and even visited profiles of many other users in my org and they all have the same problem. I don't know when it will be resolved, but it looks awkward and frustrating at the same time.

**Comment by sklm** [comment]
Try disable the uBlock Origin, or other blocker you may be using, on the GitHub website.

**Comment by rd-universe** [comment]
When I look at the page now, it appears normal. Can you tell me how you solved it?
